web/impact/impact/v1/helpers/profile_helper.py

```diff
@@ -29,23 +29,9 @@
 )
 
 
-EXPERT_DESCRIPTION = "This field exists only when user_type is 'expert'"
-EXPERT_BOOLEAN_FIELD = {
-    "json-schema": {
-        "type": "boolean",
-    },
-    "GET": {
-        "included": "could_be_expert",
-        "description": EXPERT_DESCRIPTION,
-    },
-    "PATCH": {"required": False, "allowed": "is_expert"},
-    "POST": {"required": False, "allowed": "could_be_expert"},
-}
+EXPERT_DESCRIPTION = "This field exists only when user_type is 'expert'."
 
-EXPERT_STRING_FIELD = {
-    "json-schema": {
-        "type": "string",
-    },
+OPTIONAL_EXPERT_FIELD = {
```

Comment:
"could_be_expert" and "is_expert" are repeated several times. Should be constants?

Reply:
Fixed.